src/lib/Preview.js

```diff
@@ -204,10 +204,8 @@ class Preview extends EventEmitter {
 
                 this.emitLogEvent(PREVIEW_METRIC, {
                     event_name: PREVIEW_END_EVENT,
-                    value: {
-                        duration: previewDuration,
-                        viewer_status: this.viewer.getLoadStatus(),
-                    },
+                    value: previewDuration,
+                    viewer_status: this.viewer.getLoadStatus(),
```

Comment:
I don't think this will work because that field doesn't exist in the metrics table. You'll need to emit a separate event entirely